coldfront/templates/email/allocation_revoked.txt

```diff
@@ -0,0 +1,10 @@
+Dear {{center_name}} user,
+
+Your allocation request for {{resource}} has been revoked.
```

Comment:
```suggestion
Your allocation for {{resource}} has been revoked.
```
Let's remove "request" since we're only revoking an existing, likely active, allocation, not a new allocation or renewal request.